databricks/koalas/series.py

```diff
@@ -94,11 +94,11 @@ class Series(_Frame):
     """
 
     @derived_from(pd.Series)
-    def __init__(self, data=None, index=None, dtype=None, name=None, copy=False, fastpath=False):
+    def __init__(self, data=None, index=None, dtype=None, name=None, copy=False, fastpath=False, anchor=None):
         if isinstance(data, pd.Series):
             self._init_from_pandas(data)
         elif isinstance(data, spark.Column):
-            self._init_from_spark(data, index, dtype)
+            self._init_from_spark(data, anchor, index)
```

Comment:
I noticed that we currently reassign different values and different types to the arguments of the dataframe. This is not the spirit and or recommended style in python. Instead of having a complex mechanism for the constructor, it is much simpler to have a function called `Series` in `namespace.py`.